BankSDK/GiniBankSDKExample/GiniBankSDKExample.xcodeproj/project.pbxproj

```diff
@@ -218,6 +247,13 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
+		F4C723A728F8354400BDDEA7 /* Frameworks */ = {
+			isa = PBXFrameworksBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
```

Comment:
[clarification]: I don't know exactly if we can remove this since is empty and no framework was added.

Reply:
I didn't add anything here, Xcode generated it like that.